src/test/java/org/pesmypetcare/webservice/dao/medalmanager/MedalDaoTest.java

```diff
@@ -50,17 +55,19 @@
 
     @BeforeEach
     public void setUp() {
-        medalEntity = new MedalEntity("Walker", new ArrayList<>(), "You have to walk a lot!");
+        medal = new Medal();
+        medal.setName("Walker");
+        medal.setDescription("You have to walk a lot!");
+        medal.setLevels(new ArrayList<>(Arrays.asList(5., 10., 25., 50., 100.)));
+        medal.setMedalIconPath(new byte[] {10, 55, 67 ,89, 103, 116});
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['10' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5623946)